src/engraving/infrastructure/internal/engravingconfiguration.cpp

```diff
@@ -149,22 +149,22 @@ double EngravingConfiguration::guiScaling() const
     return uiConfiguration()->guiScaling();
 }
 
-Color EngravingConfiguration::selectionColor(int voice) const
+Color EngravingConfiguration::selectionColor(engraving::voice_idx_t voice) const
```

Comment:
why that `engraving::` here but not further down?

Reply:
removed